documentation/Functions.md

```diff
@@ -297,18 +297,32 @@ will run 'npm install' in the directory 'my-thai-star/angular'
 ***
 
 ### dockerCompose <a name="dockerCompose"></a>
+This function is used for docker compose.
```

Comment:
This function is used to running [Docker Compose](https://docs.docker.com/compose/).